dockstore-client/src/main/java/io/dockstore/client/cli/nested/AbstractEntryClient.java

```diff
@@ -1108,7 +1108,25 @@ private void processWesCommands(final List<String> args) {
     }
 
     /**
-     * Launches tools and workflows.
+     * Prints a warning if Docker isn't running. Docker is not always needed. If a workflow or tool uses Docker and
+     * it is not running, it fails with a cryptic error. This should make the problem more obvious.
+     */
+    void checkIfDockerRunning() {
+        ProcessBuilder pb = new ProcessBuilder("docker", "ps");
+        try {
+            Process process = pb.start(); // run a bash command
+            process.waitFor();
+            if (process.exitValue() != 0) {
+                String type = this.getEntryType().toLowerCase(); // "tool" or "workflow"
+                out("WARNING: Docker is not running. If this " + type + " uses Docker, it will fail.");
+            }
+        } catch (IOException | InterruptedException e) {
+            exceptionMessage(e, "Error checking for Docker.", 1);
```

Comment:
I don't think you should exit if the check fails (`exceptionMessage()` does a `System.exit`). I think you should just silently ignore this error and continue.

Reply:
Replaced exception with logging the error and continuing